api/bolt/migrator/migrate_dbversion27.go

```diff
@@ -0,0 +1,11 @@
+package migrator
+
+func (m *Migrator) updateSettingsToDB28() error {
```

Comment:
@sbusso please confirm `DB28` is the right version number here

Reply:
@ArrisLee we need to create a new migration and use the reserved version number from https://portainer.atlassian.net/wiki/spaces/TECH/pages/132284689/Migrations (31)

Reply:
@sbusso updated